### PR TITLE
added documentation for monoprice_snapshot/monoprice_restore services.

### PR DESCRIPTION
**Pull request by etsinko**
**Description:**

Documentation for new services for `monoprice` platform.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#10296

### CODE REVIEW

**Comment by frenck** [comment]
@etsinko, Thank you for this PR. Looks awesome! 🏅 
Can be merged as soon as the parent PR is merged.

File: source/_components/media_player.monoprice.markdown [hunk 1]
```diff
@@ -50,16 +50,22 @@ Configuration variables:
...
 
 ### {% linkable_title Service `monoprice_restore` %}
 
-Restore a previously taken snapshot of one or more speakers. If no `entity_id` is provided, all speakers are restored.
+Restore a previously taken snapshot of one or more speakers. If no `entity_id` is provided, all zones are restored.
 
 | Service data attribute | Optional | Description |
 | ---------------------- | -------- | ----------- |
-| `entity_id` | yes | String or list of strings that point at `entity_id`s of coordinator speakers.
+| `entity_id` | yes | String or list of strings that point at `entity_id`s of zone.
```
Comment (frenck) [inline source/_components/media_player.monoprice.markdown]: Typo: `zone` -> `zones`

**Comment by frenck** [comment]
@etsinko Found a minor typo, could you please take a look?

**Comment by etsinko** [comment]
@frenck Done!

**Comment by etsinko** [comment]
@frenck the parent PR has been merged.

**Comment by frenck** [comment]
@etsinko Thank you for the ping. Going to merge this right now.

**Pull request merged**